comb/repo/devshells.nix

```diff
@@ -0,0 +1,21 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) std;
+  lib = inputs.nixpkgs.lib // builtins;
+in
+  lib.mapAttrs (_: std.lib.dev.mkShell) {
+    default = {
+      name = "CreateCommands";
+      nixago = with cell.configs;
+        [
+          treefmt
+        ]
+        ++ (with inputs.std-data-collection.data.configs; [
+          editorconfig
+          conform
+          lefthook
```

Comment:
I don't know what conform and lefthook is but it doesn't seem relevant to this repo, at least not at the moment.

I'm normally a large proponent of editorconfig but: I'd prefer a handwritten one that only cares about .nix and maybe .md, and it might be redundant if there's a formatter.

If we get rid of editorconfig, conform and lefthook here it means one less flake input which to me is a win.